.gitlab/ci/instance-test.yml

```
@@ -13,7 +13,7 @@ test_instances:
   script:
     - echo "======= Download configuration ========"
     - ./Tests/scripts/download_demisto_conf.sh  >> $ARTIFACTS_FOLDER/logs/installations.log
-    - mv $SSH_CONFIGURATION ~/.ssh/config
+    - cat $SSH_CONFIGURATION >> ~/.ssh/config
```

🍬 !!! Great catch!
```suggestion
    - cp $SSH_CONFIGURATION ~/.ssh/config
```
Or you can leave it as you have it here. Whichever you think is better.

---

The cp doesn't append, it overwrites.  >> does